scripts/replace-legacy-export.cjs

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+'use strict'
+
+const { readFileSync, writeFileSync } = require('fs')
+
+// Cleanup the export statement in CJS typings file generated by rollup:
+const legacyTypings = 'build/index.cjs.d.ts'
```

Comment:
I'm very leery of implicit path locations, silently dependent on "where" one executes the code from.

```suggestion
const legacyTypings = require('path').resolve(__dirname, '../build/index.cjs.d.ts')
```